nodejs/package.json

```diff
@@ -19,9 +19,6 @@
         "test": "mocha test/*.js",
         "buildProto": "pbjs -t static-module -w commonjs -o gtfs-realtime.js ../gtfs-realtime.proto && pbts -o gtfs-realtime.d.ts gtfs-realtime.js"
     },
-    "engines": {
-        "node": "18.12.1"
```

Comment:
Why not use a range here, e.g. `>= 18.12` or `>= 18.12 <19`?

Reply:
@derhuerst for this initial PR, I want to revert back to the original state where we didn't introduce any version restrictions. Setting a range like `>= 18.12` would cause any newly published version to be incompatible with any node versions outside the range, which I don't want to introduce haphazardly (since it would force devs to either upgrade their node version OR be stuck on an older version of this package).

My preference would be to merge this PR and then for us to have a more intentional discussion around what minimum version might make the most sense to balance compatibility with stability